app/uk/gov/hmrc/taxenrolmentassignmentfrontend/models/AccountDetails.scala

```diff
@@ -69,6 +71,8 @@ case class AccountDetails(
           .format(DateTimeFormatter.ofPattern("a"))
           .toUpperCase // <- Older versions of Java return this in lower case
     }
+
+  def isIdentityProviderOneLogin: Boolean = identityProviderType == ONE_LOGIN
```

Comment:
this does not sound right?

Reply:
It's done the same way in UserGroupResponse, I could do a .toString or something but seems redundant
https://github.com/hmrc/tax-enrolment-assignment-frontend/blob/5fc56919b3772a51172b3185b289e11b20572b56/app/uk/gov/hmrc/taxenrolmentassignmentfrontend/models/UsersGroupResponse.scala#L35

Reply:
Is accountDetails only holding One login details? No GG details?

Reply:
misread the equality with assignment. All good....